iterators_task_1/student/include/impl.h

```diff
@@ -10,4 +12,28 @@
  *
  * @warning YOU MUST USE STRING STREAMS AND STREAM ITERATORS!
  */
-std::string reverseWords(const std::string& input);
+std::string reverseWords(const std::string& input)
+{
+    std::vector<std::string> token;
+    std::istringstream in(input);
+    std::ostringstream out;
+    
+    std::move(
```

Comment:
no profit from move here
consider:
stream data: "abc"
you're adding "abd"
how can we move "abd" to "abc"? no way, we must copy